doc/rst/platforms/cray.rst

```diff
@@ -144,20 +150,22 @@ Building Chapel for a Cray System from Source
 3) Select the target compiler that Chapel should use when compiling
    code for the compute node:
 
-   On a Cray CS series system, set the ``CHPL_TARGET_COMPILER`` environment
-   variable to indicate which compiler to use (and make sure that the compiler
-   is in your path).
+   On an HPE Apollo or Cray CS series system, set the
+   ``CHPL_TARGET_COMPILER`` environment variable to indicate which
+   compiler to use (and make sure that the compiler is in your path).
 
       ===========================  ==============================
       To request...                set CHPL_TARGET_COMPILER to...
       ===========================  ==============================
-      ...the GNU compiler (gcc)    gnu    (default)
+      ...the LLVM/clang backend    llvm (default)
+      ...the GNU compiler (gcc)    gnu
+      ...the Clang compiler        clang
       ...the Intel compiler (icc)  intel
       ===========================  ==============================
 
-   On a Cray XC or HPE Cray EX system, ensure that you have one of the
-   following Programming Environment modules loaded to specify your
-   target compiler::
+   On an HPE Cray EX or Cray XC system, when using the C back-end,
+   ensure that you have one of the following Programming Environment
+   modules loaded to specify your target compiler::
```

Comment:
similar to above, should the `PrgEnv-gnu` come before `PrgEnv-cray` in this list?

Reply:
Hmm, good question.  I reordered above because gnu is definitely preferred between the two.  This list is arguably alphabetized, which might be justifiable, esp. since I'm not sure how to sort allinea and intel (nor sure why they weren't mentioned earlier).  This whole section feels also feels a little outdated/overkill given that llvm is the preferred default back-end now.  I think I might just leave this alphabetized for those reasons (i.e., "lack of investment in the messaging here given the prominence of LLVM").  If I had more energy or time, I might try to more majorly rework this section to talk about llvm being the preferred back-end.

Reply:
Ah I did not catch that this was alphabetized and not in priority order. Given LLVM is preferred anyways, I agree with leaving it as-is